waddrmgr/scoped_manager.go

```diff
@@ -1636,6 +1639,117 @@ func (s *ScopedKeyManager) ImportScript(ns walletdb.ReadWriteBucket,
 	return scriptAddr, nil
 }
 
+// ImportWitnessScript imports a user-provided script into the address manager.  The
+// imported script will act as a pay-to-witness-script-hash address.
+//
+// All imported script addresses will be part of the account defined by the
+// ImportedAddrAccount constant.
+//
+// When the address manager is watching-only, the script itself will not be
```

Comment:
See comment above about this sentence. Or maybe we can add a switch to decide whether this is considered to be _that_ private? So the user of the library can choose what crypto key to use? cc @Roasbeef